core.py

```diff
@@ -14,6 +15,28 @@ def core_addition(input_object: str) -> str:
     
     data_dict = typeconversion.json_to_dict(input_object)
     result = calculations.addition(data_dict["number1"], data_dict["number2"])
-    output_object = typeconversion.dict_to_json(result)
+    
+    return typeconversion.dict_to_json(result)
+
 
-    return output_object
+def core_component_create_edit(input_object: str) -> str:
```

Comment:
Könnt ihr nochmal bestätigen, dass hier wirklich ein JSON object übergeben wird und kein dict? Wir haben bei addition bemerkt, dass es da wohl durch den Aufruf von endpoints.py schon zu einer Umwandlung in ein dict automatisch kam. Ist das hier auch so oder hatten wir nur einen Bug irgendwo? Ansonsten wäe ich auch mit 

```suggestion
def core_component_create_edit(input_object: Union[Dict[str, Any], str]) -> str:
```
fein, falls das für euch Sinn ergibt. (sodass beide Fälle abgefangen werden)
typeconversion.json_to_dict kann ja mit beidem umgehen.

Reply:
Wir warten noch auf die fertigen fkt vom databasehandler und testen das dann :)

Reply:
Okay super danke